src/utils/periods/relativePeriods.js

```diff
@@ -1,179 +1,224 @@
 import i18n from '@dhis2/d2-i18n'
 import { fixedPeriodTranslations } from './fixedPeriods'
 
+const RELATIVE_PERIOD_CODES = {
```

Comment:
Any reason you went with an object here instead of string constants? I think string constants would give more certainty in that you can be sure that they won't be overwritten. Which feels appropriate for this usecase.

Reply:
Can't remember why I ended up with those. I gradually moved stuff around and refactored. Perhaps at some point in time these codes were needed in a different file and therefore it made more sense to export them as an object. In the current state of the codebase, your suggestion makes perfect sense though. Implemented in b9329cd